docs/data/pages.ts

```diff
@@ -2,8 +2,7 @@ import type { MuiPage } from '@mui/monorepo/docs/src/MuiPage';
 
 const pages: MuiPage[] = [
   {
-    pathname: '/x/introduction',
-    scopePathnames: ['/x/introduction'],
+    pathname: '/x/introduction-group',
```

Comment:
```suggestion
    pathname: '/x/introduction-group',
    title: 'Introduction',
```
To avoid a regression on the side menu section title, which changed from `Introduction` to `Introduction group` 🙈